editions/free/src/css/lobby.css

```diff
@@ -67,8 +67,8 @@ div.frame {
 
 #topbar ul.topbar-nav .logo .logo-icon {
     display: block;
-    width: ${css_vh(28.65)};
-    height: ${css_vh(10.03)};
+    width: ${css_vh(10)};
+    height: ${css_vh(10)};
     background: url('../assets/lobby/scratchJrlogo.svg');
```

Comment:
You need to rename the file to something that makes sense.
You could use something like 'prevPage'

background: url('../assets/lobby/<new_name_of_file>');